python/packages/isce3/math/__init__.py

```diff
@@ -1,2 +1,3 @@
 from isce3.ext.isce3.math import *
 from .snap import snap, snap_datetime
+from .offsets_polyfit import polyfit_offsets, predict_offsets_batch, build_design_matrix, ncoeffs, normalize
```

Comment:
```suggestion
from . import offsets_polyfit
```
I don't think we should import all these names into `isce3.math`.